### PR TITLE
Update icon

### PR DESCRIPTION
**Pull request by philipwhiuk**
Use the new icon both on the main page and in the favicon.

### CODE REVIEW

**Comment by cketti** [comment]
The dimensions of the old and the new favicon don't match.

There are a bunch of other image files under `assets/ico/`. If we don't need them anymore we should delete them. If we do need them, they should be changed to the new icon as well.

**Comment by philipwhiuk** [comment]
The size doesn't seem to be that important (https://stackoverflow.com/questions/2268204/favicon-dimensions) - I just picked the smallest that was in the original PR. I'll try and see if there's 32x32 version though.

I'll take a look at the ico files.

File: assets/ico/manifest.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+ "lang": "en",
+ "dir": "ltr",
+ "name": "K-9 Mail Website",
+ "start_url": ".../../index.html",
```
Comment (cketti) [inline assets/ico/manifest.json]: `...` doesn't seem right.

File: assets/ico/manifest.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ "description": "Information about K-9, an open source email client",
+ "short_name": "K9Mail",
```
Comment (cketti) [inline assets/ico/manifest.json]: "K-9 Mail". If it needs to be a string without "special" characters I'd prefer "k9mail".

File: assets/ico/manifest.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+ "lang": "en",
+ "dir": "ltr",
+ "name": "K-9 Mail Website",
+ "start_url": ".../../index.html",
+ "description": "Information about K-9, an open source email client",
```
Comment (cketti) [inline assets/ico/manifest.json]: Please don't abbreviate to K-9, but use "K-9 Mail".

File: assets/ico/manifest.json (hunk 2 above)
Comment (philipwhiuk) [reply assets/ico/manifest.json]: The example did that but I'm not sure it's necessary. Sticking with the full name for now.

**Pull request merged**